TINY-9504: Fix clicking on a disabled split button (#8409)
tinymce/tinymce: modules/tinymce/CHANGELOG.md

- Removed a workaround for ensuring stylesheets are loaded in an outdated version of webkit. #TINY-9433
- Lists in a noneditable root were incorrectly editable using list API commands, toolbar buttons and menu items. #TINY-9458
- Color picker dialog would not update the preview color if the hex input value was prefixed with `#` symbol. #TINY-9457

> - Clicking on a disabled split button will not call `onAction` callback anymore. #TINY-9504

Reviewer: Just a slight wording tweak

```suggestion
- Clicking on a disabled split button will no longer call the `onAction` callback. #TINY-9504
```
